Tasks/Npm/npm.ts

```diff
@@ -10,16 +11,18 @@ import * as npmCustom from './npmcustom';
 import * as npmPublish from './npmpublish';
 import { GetRegistries, NormalizeRegistry } from 'npm-common/npmrcparser';
 import { INpmRegistry, NpmRegistry } from 'npm-common/npmregistry';
+import * as telemetry from 'utility-common/telemetry';
 import { NpmToolRunner } from './npmtoolrunner';
 import * as util from 'npm-common/util';
 
 async function main(): Promise<void> {
     tl.setResourcePath(path.join(__dirname, 'task.json'));
+    await _logNpmStartupVariables();
```

Comment:
I would log this in the end, that way we can gather more interesting data points along the task's execution and log them all at once.

Reply:
This was intentional.  I care about seeing all of the values before the task does anything.  In case the task suddenly exits.  IOW, think of this as an "entry log".  There is no "exit log" and I don't plan on adding one at this time.  I _may_ revisit this if I feel like I need more telem.